install.sh

```diff
@@ -495,40 +506,17 @@ function configure_installation_variables() {
 	fi
 }
 
-function configure_gateway_node() {
```

Comment:
There is still a reference to gateway labels in the `uninstall_liqo` function, which should be probably dropped.